tests/utils.py

```diff
@@ -74,3 +78,16 @@ def print_and_test(expected, result=None, actual=None):
     print("\n".join([" - ".join(v) for v in get_differences(expected, result)]))
 
     assert result == expected
+
+
+def get_mock_bq_client(schema: List[SchemaField]):
```

Comment:
👍 Can you update tests/test_glean_ping_view.py to use this as well (fine if it's in a seperate PR)?

Reply:
Yup, I'll add it to the backlog of refactoring 😄  My goal in general is to make it simpler to add new generated views and explores. We're pretty close to having it just be a few new files rather than changing a bunch of existing ones